tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -275,9 +275,17 @@ impl<C, N, E, W> Subscriber<C, N, E, W> {
     }
 
     /// Enable ANSI terminal colors for formatted output.
-    #[cfg(feature = "ansi")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "ansi")))]
     pub fn with_ansi(self, ansi: bool) -> Self {
+        #[cfg(not(feature = "ansi"))]
+        if ansi {
+            const ERROR: &str =
+                "tracing-subscriber: enabled ANSI terminal colors without the `ansi` crate feature";
```

Comment:
This error message describes what happened, rather than _why_ it happened, which might be more helpful for the user. How about rephrasing it as something like this:
```suggestion
                "tracing-subscriber: the 'ansi' crate feature is required to enable ANSI terminal colors";
```

Reply:
That's way better indeed.

I found that in other parts of the documentation `tracing` is using `` ` `` instead of `'` with crate features, so I used it here too. WDYT?